src/utils.ts

```diff
@@ -1,8 +1,10 @@
-let _process, getNanoSeconds, loadTime
+type HrTimeFn = () => [number, number]
+type _Process = { hrtime: HrTimeFn }
+let _process: _Process, getNanoSeconds: (() => number) | undefined, loadTime: number | undefined
 try { _process = eval('typeof __TEST_WEB__ === "undefined" && typeof process === "object" ? process : undefined') } catch (e) {} // eslint-disable-line no-eval
 
 const hasProcessHrtime = () => {
-  return (typeof _process !== 'undefined' && _process !== null) && _process.hrtime
+  return (typeof _process !== 'undefined' && _process !== null) 
```

Comment:
&& _process.hrtime !== undefined

Reply:
This can just be `return _process?.hrtime != null`.

Reply:
True, but I have been considering that in this PR I should aim to do _minimal_ changes - to not introduce any bugs or fix too many... "features"